### PR TITLE
openai, not expose getAudioTranscriptionAsPlainText getAudioTranslationAsPlainText client method

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
Personally I think we should not expose these "##AsPlainText" API

diff https://apiview.dev/Assemblies/Review/0b875c11102a447c8807ad710b3d4e7c/2b4c7ce6218d433cba5909dcaffbbeea?diffRevisionId=296f330f8a7a4abbb144e5101e3cb28b&doc=False&diffOnly=True

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java [hunk 1]
```diff
@@ -1286,9 +1286,8 @@ public Mono<Response<String>> getAudioTranslationTextWithResponse(String deploym
      * @return transcribed text and associated metadata from provided spoken audio data along with {@link Response} on
      * successful completion of {@link Mono}.
      */
-    @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Response<BinaryData>> getAudioTranscriptionAsPlainTextWithResponse(String deploymentOrModelName,
+    Mono<Response<BinaryData>> getAudioTranscriptionAsPlainTextWithResponse(String deploymentOrModelName,
```
Comment (mssfang) [inline sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java:1290]: Instead of manually remove the generated annotation, do you know why it was generated initially? It seems a possible to not regenerate this `AsPlainText` methods instead of manually remove the generated.

Comment (weidongxu-microsoft) [reply sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java:1290]: This was due to the `MultipartFormDataRequestHeadersTraits` in typespec not having effect on `Azure.Core.Foundations.ResourceOperation` template.

I was attempted to address this in spec PR https://github.com/Azure/azure-rest-api-specs/pull/26412 but didn't get response from Travis (mostly to clarify whether the `content-type` header in response be `text/plain`).

Sure, let me try again if I can revive that spec PR.

**Comment by weidongxu-microsoft** [comment]
spec PR merged https://github.com/Azure/azure-rest-api-specs/pull/26412
